col/src/main/java/vct/col/ast/print/JavaPrinter.java

```diff
@@ -1440,17 +1462,17 @@ public void visit(VariableDeclaration decl){
       sep=",";
       if (dd instanceof DeclarationStatement){
         DeclarationStatement d = (DeclarationStatement)dd;
-        d.getType().accept(this);
+        out.print(d.name());
```

Comment:
The type of the variable declaration is now no longer printed, are you sure that is correct? Or is the "decl.basetype.accept(this)" line above now responsible for printing the type?

Reply:
> The type of the variable declaration is now no longer printed, are you sure that is correct? Or is the "decl.basetype.accept(this)" line above now responsible for printing the type?

Yes "decl.basetype.accept(this)" is now responsible for printing the type.